docs/kb/semgrep-ci/github-upload-findings-in-security-dashboard.md

```diff
@@ -11,9 +11,9 @@ When scanning with Semgrep in CI, findings automatically populate in Semgrep Clo
 
 If you run the alternate job and it fails with a "resource not accessible by integration" error, there are two possible causes.
 
-## Your repository is private
```

Comment:
With these changes, I think it would make sense to reverse the order of the possible issues, since one applies to all repos, and one only applies to private repos. Since this one has greater need the config file as a concrete example, that also would allow it to be adjacent to the config file without needing to repeat the file contents, while the other can perhaps have an anchor link or something in the info box.

This is the change that's a little harder to picture/share an exact outcome for since it requires a re-jig of the structure, but hopefully my description makes sense.

As part of that, I think it would make sense to change the title of this section to "The workflow or job does not have read permissions in a private repository" or something more specific like that - otherwise, both titles reference permissions so it's not entirely clear why they're separate.